parcels/particleset/particlesetsoa.py

```diff
@@ -273,6 +316,14 @@ def error_particles(self):
         error_indices = self.data_indices('state', [StateCode.Success, StateCode.Evaluate], invert=True)
         return ParticleCollectionIterableSOA(self._collection, subset=error_indices)
 
+    def active_particles_mask(self, time, dt):
+        # TODO: make this into a mask
+        active_indices = (time - self._collection.data['time'])/dt >= 0
+        non_err_indices = np.isin(self._collection.data['state'], [StateCode.Success, StateCode.Evaluate])
+        active_indices = np.logical_and(active_indices, non_err_indices)
+        self._active_particle_idx = np.where(active_indices)[0]
+        return active_indices
```

Comment:
Okay, so THAT is the meaning of the `_active_idx`. I'd say, as this is really quite hidden, to add a short comment in the basic neighbor search class (or where that member variable is defined) about like `# '_active_idx' are indices of all particles (i) already started at the current time and (ii) are set to a positive state (i.e. Success or Evaluate)`

Reply:
I have added this description to the neighbor base class as I said in another comment.